flink-table/flink-table-common/src/main/java/org/apache/flink/table/descriptors/DescriptorProperties.java

```diff
@@ -221,6 +225,18 @@ public void putTableSchema(String key, TableSchema schema) {
 		}
 	}
 
+	/**
+	 * Adds table partition keys.
+	 */
+	public void putPartitionKeys(List<String> keys) {
```

Comment:
I am not 100% sure if we need this method. This is a very local method. And actually the only thing that this method does compared to `putIndexedFixedProperties` is that it uses a fixed `PARTITION_KEYS` for the key.

But I am fine with having it here.

Reply:
If PartitionKeys and schema are equal situation, this method will be better and more convenient.